Sources/JavaScriptKit/JSValueConvertible.swift

```diff
@@ -194,8 +194,6 @@ extension RawJSValue: JSValueConvertible {
             return .undefined
         case .function:
             return .function(JSFunction(id: UInt32(payload1)))
-        default:
-            fatalError("unreachable")
```

Comment:
What happens if this is passed an invalid `kind`?

Reply:
The swift runtime will crash with error message that says the given enum value has invalid raw value.